pyincore_data/censusviz.py

```diff
@@ -12,9 +12,9 @@
 
 from ipyleaflet import projections
 from branca.colormap import linear
-from pyincore_data import globals
+from pyincore_data import globals as pyincore_globals
 
-logger = globals.LOGGER
+logger = pyincore_globals.LOGGER
```

Comment:
Does sphinx not like the "global"?

Reply:
there is a python global and it confused both